plugin/adsk/plugin/CMakeLists.txt

```diff
@@ -50,12 +50,10 @@ target_compile_definitions(${TARGET_NAME}
         MAYAUSD_PLUGIN_EXPORT
 )
 
-if(UFE_FOUND)
-    target_compile_definitions(${TARGET_NAME} 
-        PRIVATE
-            -DWANT_UFE_BUILD
-    )
-endif()
+target_compile_definitions(${TARGET_NAME} 
+    PRIVATE
+        $<$<BOOL:${UFE_FOUND}>:WANT_UFE_BUILD>
```

Comment:
@seando-adsk I used generator expression $<BOOL:...> to convert FALSE or TRUE to 0 or 1

`$<UFE_FOUND:WANT_UFE_BUILD> --> this would cause an error because ${UFE_FOUND} isn't 1 or 0`

`$<$<BOOL:${UFE_FOUND}>:WANT_UFE_BUILD>   --> BOOL:${UFE_FOUND} converts TRUE or FALSE to 1 or 0`

Reply:
So the whole point was to merge this into the one above at line 48. Just move line 55 to just after 50. Then you don't need two target_compile_definitions. Same for the target_include_directories below.

Reply:
Ugh d’accord ! je suis aveugle...

Reply:
Excellent thanks.